packages/nodes-base/credentials/CrowdDevApi.credentials.ts

```diff
@@ -0,0 +1,64 @@
+import type {
+	IAuthenticateGeneric,
+	ICredentialTestRequest,
+	ICredentialType,
+	INodeProperties,
+} from 'n8n-workflow';
+
+export class CrowdDevApi implements ICredentialType {
+	name = 'crowdDevApi';
+
+	displayName = 'crowd.dev API';
+
+	documentationUrl = 'https://docs.crowd.dev/reference/getting-started-with-crowd-dev-api';
+
+	properties: INodeProperties[] = [
+		{
+			displayName: 'Domain',
+			name: 'domain',
```

Comment:
Should probably be more "url" and not "domain" as it does include "https"

Reply:
Would also add an option for self signed certificates as there will no doubt be a could of users not using Let's Encrypt or similar.